README.md

```diff
@@ -61,20 +61,21 @@ A template of the `licenses.json` file can be found in the repository at `src/ma
    # Used license scanner: scanoss (at the moment, only this scanner is supported)
     scanner=scanoss
 
-   # Path to the 'licenses.json' file which contains information about permitted,
-   # restricted and prohibited licenses. This file should be filled according to
-   # the template which could be found at 'src/main/resources/licenses.json'
-    license_filepath=
-
    # Used license conflicts source:
-   # > option "json": take conflicts from 'licenses.json' (should be filled manually
-   # according to the template at 'src/main/resources/licenses.json')
+   # > option "db": take conflicts from MySQL database - 'license_conflicts' table (should be filled manually
```

Comment:
Currently we do not consider such option. Will add it in future releases after the release 1.0.2 in February.

Reply:
LPVS can already be run with [the `scanner` option](https://github.com/Samsung/LPVS/blob/783b904e1c212d220a317e24b12f2956cec01617/src/main/resources/application.properties#L13). 
I wanted to see if LPVS could run without MySQL.

Reply:
In case we disable DB with license info, scanner can only identify the license of used component and identify license conflicts if 2 or more licenses are presented.
Identification of license type (permitted, restricted or prohibited) will not be available.